src/components/posts/style.css

```diff
@@ -1,34 +1,39 @@
 .home-container {
-  display: flex;
-  justify-content: space-evenly;
-  padding-top: 2.5rem;
+    display: flex;
+    justify-content: space-evenly;
+    padding-top: 2.5rem;
 }
 
 .posts-section {
-  flex: 1;
-  margin-top: 15px;
+    margin-top: 15px;
```

Comment:
Watch formatting and spacing in here, looks different from the rest of the file. I belive the convention is 2 spaces for tabs.